backend/server.py

```
@@ -1501,27 +1501,16 @@ async def lifespan(app_: "FastAPI"):
+    # drops immediately after a deploy (the cron tick may not reach a sleeping
+    # Render free-tier instance for several minutes).
     try:
-        import asyncio
-        asyncio.create_task(_deferred_startup_cleanup())
-    except Exception as exc:  # pragma: no cover - defensive
-        log.warning("Could not schedule deferred startup cleanup: %s", exc)
+        cleanup = await SCHEDULER.force_cleanup()
+        if cleanup.get("deleted", 0) > 0 or cleanup.get("deduped", 0) > 0:
+            log.info("Startup schedule cleanup: deleted=%d deduped=%d total=%d",
+                     cleanup["deleted"], cleanup["deduped"], cleanup["total"])
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)